client-src/elements/chromedash-app.js

```diff
@@ -138,6 +138,17 @@ class ChromedashApp extends LitElement {
   setUpRoutes() {
     page.strict(true); // Be precise about trailing slashes in routes.
 
+    const scrollIfHash = (ctx) => {
+      // If current page is ctx.path and only the hash changes, don't create a new element
+      // but instead just scroll to the element identified by the hash.
+      if (this.currentPage == ctx.path && ctx.hash) {
```

Comment:
I copied the code used on the /metrics/:type/:view page, but maybe the query string is never used there?  I suppose we could check that the query string is unchanged as well, but is that enough?  Alternatively, is it even necessary if we don't use the query string on the guide pages?  

Here is a possible problem case: If the query string changes but the path does not change, and there is a hash, then it will jump to the hash and ignore the query string change.

Since we don't appear to be using the query string on the guide pages, I'll just add a comment in this function about ignoring the query string.